packages/components/avatar/src/avatar.tsx

```diff
@@ -94,7 +92,11 @@ const GravatarImg = (props: TGravatarImgProps) => (
       `${createGravatarImgUrl(props.gravatarHash, props.size)} 1x`,
       `${createGravatarImgUrl(props.gravatarHash, props.size, 2)} 2x`,
     ].join(',')}
-    alt={props.altText}
+    /**
+     * Based on https://www.w3.org/WAI/tutorials/images/decision-tree/:
+     * image contributes meaning to the current page or context and it shows content that is redundant to real text nearby
+     */
+    alt=""
```

Comment:
This might be a little controversial, so I wanted to give a little context.
The issue is reported in axe and this is the [writeup](https://dequeuniversity.com/rules/axe/4.4/image-redundant-alt?application=AxeChrome) referenced by axe.
 
My solution was based on the [alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/). 
<img width="883" alt="Screenshot 2022-07-01 at 12 36 17" src="https://user-images.githubusercontent.com/49066275/176883217-64213a85-55df-4e9a-8617-30ab862d7a3d.png">

Additionally, please note that this is what e.g. Google does:
<img width="677" alt="Screenshot 2022-07-01 at 12 39 28" src="https://user-images.githubusercontent.com/49066275/176883376-d2f8d782-681c-4b7f-806d-15424f789ba5.png">

On the other hand, I have also found many other solutions e.g.:
- Atlassian's [avatar component](https://atlassian.design/components/avatar/examples) contains full name as alt.
- GitHub, depending on the page, uses `alt=“”`, `alt=“Avatar”` or `alt=“@username”` 🤯

It would be cool to know your opinions.

Reply:
Thanks for looking into that!

I think using `alt=""` or `alt="Avatar"` would be my preference.

Reply:
I also agree having an empty `alt` is a good choice here 👍.